### ISSUE TITLE
loss = nan

### ISSUE THREAD

**Issue by Be997398715** [GodsDusk/FashionAI-keypoints#1]
I use your code to train,but the loss is always nan, is it right?

**Comment by GodsDusk** [comment]
> I use your code to train,but the loss is always nan, is it right?

In my experiments，lr <= 1e-5 will cause nan， and 1e-3 runs fine

**Comment by Be997398715** [comment]
Now I use 1e-3，but I don't know if I should run 800000 steps . if I use 1e-3 for 100000 steps and then use 1e-5 for left steps, does it work?

**Comment by GodsDusk** [comment]
> Now I use 1e-3，but I don't know if I should run 800000 steps . if I use 1e-3 for 100000 steps and then use 1e-5 for left steps, does it work?

sure,  I use 1e-5 for finetune

**Comment by Be997398715** [comment]
Right, I've run 30000 steps, the loss is from several millions to tens of millions ,  I wanna know how much the final loss is fine ? Thank you.

**Comment by GodsDusk** [comment]
> Right, I've run 30000 steps, the loss is from several millions to tens of millions , I wanna know how much the final loss is fine ? Thank you.

I don't quite remember，assumably millions after 800000 steps， you may write your own metric during training。

**Comment by Be997398715** [comment]
Thank you! I will try my self.
